docker-compose.acceptance.yaml

```diff
@@ -0,0 +1,38 @@
+version: '3.8'
+
+services:
+  ffc-pay-processing:
+      build: 
+        target: development
+      image: ffc-pay-processing-development
+      volumes:
+        - ./app:/home/node/app
+        - ./package.json:/home/node/package.json
+  
+  ffc-pay-processing-test-runner:
+    build:
+      context: .
+      target: development
+    depends_on:
+      - ffc-pay-processing
+    image: ffc-pay-processing-development
+    command: npm run test:acceptance
+    volumes:
+      - ./test-output:/home/node/test/acceptance/test-output
```

Comment:
Should be bound to `home/node/test-output`.  Files should be written there instead to be consistent with all other test outputs.